pkg/processor/build/template.go

```diff
@@ -16,16 +16,17 @@ limitations under the License.
 
 package build
 
-const registryFileTemplate = `// Auto generated code by Nuclio
-package main
-
-import (
-	"github.com/nuclio/nuclio/pkg/processor/runtime/golang/event_handler"
-	"github.com/nuclio/nuclio/cmd/processor/user_functions/{{.Name}}"
-)
-
-func init() {
-     golangruntimeeventhandler.EventHandlers.Register("{{.Name}}", golangruntimeeventhandler.EventHandler({{.Name}}.{{.Handler}}))
-}
-// Auto generated code by Nuclio
+var processorImageDockerfileTemplate = `FROM {{baseImageName}}
+
+{{range $sourcePath, $destPath := objectsToCopy}}
+COPY {{$sourcePath}} {{$destPath}}
```

Comment:
`COPY` has a different semantics if `$sourcePath` is a directory or not.

Reply:
?